src/subcommand.go

```diff
@@ -126,8 +125,8 @@ func CommandReadRun() {
 	switch cmdName {
 	case "add":
 		cmd := flag.NewFlagSet("add", flag.ExitOnError)
-		dataTypeStr := cmd.String("type", "text", "The type of the input data")
-		sourceTypeStr := cmd.String("source-type", "local_file", "How to access the content")
+		dataType := cmd.String("type", "text", "The type of the input data")
```

Comment:
Since you changed this name here, shouldn't we change this for the other subcommands to the same thing for uniformity?

Reply:
Right. Done